### DISCUSSION TITLE
Feature Request: Open browser without prompt for matching URLs

### DISCUSSION THREAD

**Discussion by barthez** [will-stone/browserosaurus#448] [Ideas]
Hey, It is really great product and it makes my workflow much easier. I have a setup where I use Chrome for work related things and Firefox for private. I would love to have a way to define set of urls/url patterns and default browsers for them, so I could skip the prompt and have URL opened in designated browser, like my GitHub organization page or the product page opened in Chrome and memes and others opened in Firefox. WDYT?

**Comment by will-stone** [comment]
Hi @barthez thanks for using Browserosaurus. Please see the [FAQ](https://github.com/will-stone/browserosaurus/blob/master/docs/faq.md#i-would-like-to-open-a-particular-browser-automatically-for-specific-urls) for the answer.

**Comment by barthez** [comment]
Thank you for your prompt answer, that would definitely work for me!
